django/db/backends/mysql/features.py

```diff
@@ -33,6 +33,20 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_slicing_ordering_in_compound = True
     supports_index_on_text_field = False
     has_case_insensitive_like = False
+    procedure_without_params_sql = """
+        CREATE PROCEDURE test_procedure ()
+        BEGIN
+            DECLARE V_I INTEGER;
+            SET V_I = 1;
+        END;
+    //"""
```

Comment:
I'm curious if the slashes are needed. Both on MySQL and Oracle, the tests seem to pass without them.